testdata/jsight_0.3/ser-to-openapi/req.japi.http_method.query_0.3/query_user_type_allOf.openapi.todo

```diff
@@ -0,0 +1,370 @@
+{
+  "openapi": "3.0.3",
+  "info": {
+    "title": "",
+    "version": ""
+  },
+  "components": {
+    "schemas": {
+      "queryUserType1": {
+        "type": "object",
+        "properties": {
+          "first": {
+            "type": "integer",
+            "example": 123
+          },
+          "second": {
+            "type": "string",
+            "example": "value"
+          }
+        },
+        "required": [
+          "first",
+          "second"
+        ],
+        "additionalProperties": false
+      },
+      "queryUserType2": {
+        "type": "object",
+        "properties": {
+          "third": {
+            "type": "number",
+            "example": 456.78
+          },
+          "fourth": {
+            "type": "boolean",
+            "example": true
+          }
+        },
+        "required": [
+          "third",
+          "fourth"
+        ],
+        "additionalProperties": false
+      },
+      "queryUserTypeAllOf": {
+        "type": "object",
+        "properties": {
+          "fifth": {
+            "type": "string",
+            "example": "5"
+          }
+        },
+        "required": [
+          "fifth"
+        ],
+        "additionalProperties": false,
+        "allOf": [
+          {
+            "$ref": "#/components/schemas/queryUserType1"
+          },
+          {
+            "$ref": "#/components/schemas/queryUserType2"
+          }
+        ]
+      }
+    }
+  },
+  "paths": {
+    "/query/one-user-type": {
```

Comment:
In the source jsight document, there is no `/query/one-user-type` path, but two `/query/one-user-type-1` and `/query/one-user-type-2` paths.

Reply:
Fixed in `query_user_type_allOf.jst` by leaving only one  `GET /query/one-user-type`